include/ccl_power.h

```diff
@@ -6,6 +6,8 @@ extern "C" {
 
 #include "ccl_core.h"
 
+double ccl_comoving_matter_density(ccl_cosmology *cosmo);
```

Comment:
Do you need these after Shahab's modifications?

Reply:
No, not really. Is the correct thing to do to 'rebase' my branch on the now updated master branch so that I can have access to Shahab's new density functions? I tried to do this but ran into conflicts that were to do with files created by SWIG. Is there some general preferred method for rebasing, or any idea how to deal with the conflicts?

Reply:
Done now